ova/provision.sh

```diff
@@ -1,56 +1,129 @@
 #/bin/bash
+
 set -exf
-# Variables
-repo_branch=$(echo "$1" | cut -c1-3)
-repo_baseurl=$(echo "$1" | cut -c1-2)
+
 WAZUH_VERSION=$1
 OPENDISTRO_VERSION=$2
 ELK_VERSION=$3
 PACKAGES_REPOSITORY=$4
 BRANCH=$5
-UI_REVISION=$6
-DIRECTORY="/var/ossec"
-ELK_MAJOR=`echo ${ELK_VERSION}|cut -d"." -f1`
-ELK_MINOR=`echo ${ELK_VERSION}|cut -d"." -f2`
+BRANCHDOC=$6
+INSTALLER="all-in-one-installation.sh"
 
-CURRENT_PATH="$( cd $(dirname $0) ; pwd -P )"
-config_files="${CURRENT_PATH}/Config_files"
-automatic_set_ram_location="/etc/"
-libraries_files="${CURRENT_PATH}/Libraries/"
+# Display dev/prod
+echo "Using ${PACKAGES_REPOSITORY} packages"
 
-echo "${PACKAGES_REPOSITORY}"
+# Create user wazuh - Ignore password error
+adduser wazuh
+yes wazuh | sudo passwd wazuh
 
+# Grant sudo privileges to user
+gpasswd -a wazuh wheel
 
-. ${CURRENT_PATH}/Libraries/provision-opendistro.sh
+# Set Hostname
+hostname wazuhmanager
 
+# Ssh config
+sed -i "s/PasswordAuthentication no/PasswordAuthentication yes/" /etc/ssh/sshd_config
+echo "PermitRootLogin no" >> /etc/ssh/sshd_config
+
+# Download unattended installer
+curl -so ${INSTALLER} https://raw.githubusercontent.com/wazuh/wazuh-documentation/${BRANCHDOC}/resources/open-distro/unattended-installation/${INSTALLER} 
+
+# Get currents version values of installer
+CURRENT_W=$(less ${INSTALLER} | grep "WAZUH_VER=")
+CURRENT_O=$(less ${INSTALLER} | grep "OD_VER=")
+CURRENT_E=$(less ${INSTALLER} | grep "ELK_VER=")
+
+# Change wazuh and documentation repository branch
+sed -i "s/uh\/[0-9]\+\.[0-9]\+\/ex/uh\/${BRANCH}\/ex/g" ${INSTALLER}
+sed -i "s/on\/[0-9]\+\.[0-9]\+\/re/on\/${BRANCHDOC}\/re/g" ${INSTALLER}
+
+# Change versions
+sed -i "s/${CURRENT_W}/WAZUH_VER=\"${WAZUH_VERSION}\"/g" ${INSTALLER}
+sed -i "s/${CURRENT_O}/OD_VER=\"${OPENDISTRO_VERSION}\"/g" ${INSTALLER}
+sed -i "s/${CURRENT_E}/ELK_VER=\"${ELK_VERSION}\"/g" ${INSTALLER}
+
+# Change repository if dev is specified
+if [ "${PACKAGES_REPOSITORY}" = "dev" ]; then
+  sed -i "s/\[wazuh\]/\[wazuh_pre_release\]/g" ${INSTALLER}
+  sed -i "s/ngpgkey\=https\:\/\/packages\.wazuh\.com/ngpgkey\=https\:\/\/packages\-dev\.wazuh\.com/g" ${INSTALLER}
+  sed -i "s/baseurl\=https\:\/\/packages\.wazuh\.com\/4\.x/baseurl\=https\:\/\/packages\-dev\.wazuh\.com\/pre\-release/g" ${INSTALLER}
+  sed -i "s/https\:\/\/packages\.wazuh\.com\/4\.x\/ui\/kibana/https\:\/\/packages\-dev\.wazuh\.com\/pre\-release\/ui\/kibana/g" ${INSTALLER}
+  sed -i "s/wazuh_kibana-[0-9\.]\+_[0-9\.]\+/wazuh_kibana-${WAZUH_VERSION}_${ELK_VERSION}/g" ${INSTALLER}
+fi
+
+# Execute unattended installer
+sh ${INSTALLER}
+
+# Remove installer
```

Comment:
Over-commenting

Reply:
Fixed in https://github.com/wazuh/wazuh-packages/pull/746/commits/6fb50e3584df557334da525324a73a5c32ae0551